demo/src/main/java/eu/kevin/demo/extensions/StringExtensions.kt

```diff
@@ -0,0 +1,6 @@
+package eu.kevin.demo.extensions
+
+import java.math.RoundingMode
+
+fun String.toRepresentableBigDecimal() =
+    this.toBigDecimalOrNull()?.setScale(2, RoundingMode.FLOOR)?.toString()
```

Comment:
I woudn't use = here, but regular return as now it not very clear what is returned